clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+* Initialize a properties file at `src/main/resources/application-ccloud.properties` with configuration to your Confluent Cloud cluster:
+
+[source,shell]
+.application-ccloud.properties
+----
+❯ cat src/main/resources/application-ccloud.properties
+
+# topic config
+io.confluent.developer.config.topic.name=test
+io.confluent.developer.config.topic.replicas=3
+io.confluent.developer.config.topic.partitions=6
+
+# common configs
+spring.kafka.properties.ssl.endpoint.identification.algorithm=https
+spring.kafka.properties.sasl.mechanism=PLAIN
+spring.kafka.properties.request.timeout.ms=20000
+spring.kafka.properties.bootstrap.servers=bootstrap-server.confluent.cloud:9092
+spring.kafka.properties.retry.backoff.ms=500
+spring.kafka.properties.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="<API_KEY>" password="<API_SECRET>";
+spring.kafka.properties.security.protocol=SASL_SSL
+
+# Cloud SR Config
+spring.kafka.properties.basic.auth.credentials.source=USER_INFO
+spring.kafka.properties.schema.registry.basic.auth.user.info=SR_KEY:SR_SECRET
+spring.kafka.properties.schema.registry.url=https://sr.confluent.cloud
+
+# producer configuration
+spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=io.confluent.kafka.serializers.KafkaAvroSerializer
+
+# consumer configuration
+spring.kafka.consumer.group-id=java-springboot
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=io.confluent.kafka.serializers.KafkaAvroDeserializer
+----
+
+
+== Hello World!
+
+This Spring Boot application has two components - Producer (`ProducerExample.java`) and Consumer (`ConsumerExample.java`).
+Both components will be initialized during the Spring Boot application startup.
+The producer writes Kafka data to a topic in Confluent Cloud.
+Each record has a String key representing a username (e.g. `alice`) and a value of a count, formatted as Avro object
++
+[source,json]
+----
+{"namespace": "io.confluent.examples.clients.cloud",
+ "type": "record",
+ "name": "DataRecordAvro",
+ "fields": [
+     {"name": "count", "type": "long"}
+ ]
+}
+----
+
+The consumer reads the same topic from Confluent Cloud and prints data to the console.
+
+The Kafka Streams API reads the same topic from Confluent Cloud and does a stateful sum aggregation, also a rolling sum of the counts as it processes each record.
+
+. Run the producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
+
++
+[source, shell]
+----
+ # Build the application
+ $ ./gradlew clean build
+   
+ # Run the app
+ $ java -cp build/libs/java-springboot-0.0.1-SNAPSHOT.jar -Dloader.main=io.confluent.examples.clients.cloud.springboot.kafka.SpringbootKafkaApplication org.springframework.boot.loader.PropertiesLauncher <1>
```

Comment:
`<1>` (and numbers below) -- what is their purpose?  It doesn't work with them so they should be removed